src/components/KidsForm.js

```diff
@@ -0,0 +1,173 @@
+
+  const filteredName = kidsData.filter(item => item.name.toLowerCase().trim() === nameInput.toLowerCase().trim());
+  const filteredNameAndDate = kidsData.filter(
+    item =>
+      item.birthDate === startDate.toLocaleDateString('de-DE', options) &&
+      item.name.toLowerCase().trim() === nameInput.toLowerCase().trim()
+  );
+
+  const disabledButton = nameInput.length <= 1 || startDate === null || filteredNameAndDate.length > 0 ? true : false;
+
+  function handleTextInput(e) {
+    setNameInput(e.target.value);
+  }
+
+  async function onCreateKidFormSubmit(event) {
+    event.preventDefault();
+    let id = nanoid();
+    let newDate = startDate.toLocaleDateString('de-DE', options);
+
+    setKidsData([...kidsData, { id: id, name: nameInput.trim(), birthDate: newDate }]);
+    setStartDate(new Date());
+    setNameInput('');
+    setShowMessage(true);
+    navigate('/addphrases');
+  }
+
+  function handleEmptyNameOnFocusDate() {
+    if (nameInput.length === 0) {
+      setIsName(true);
+    } else {
+      setIsName(false);
+    }
+  }
+
+  return (
+    <Wrapper>
+      <FormWrapper onSubmit={onCreateKidFormSubmit}>
+        <LabelInputText htmlFor="kids-name">Wie heisst Dein Kind?</LabelInputText>
+        <TextInput
+          type="text"
+          id="kids-name"
+          name="kids-name"
+          onChange={handleTextInput}
+          onFocus={() => setIsName(false)}
+          maxLength="20"
+          placeholder="Vorname (erforderlich)"
+          autoFocus
+          autoComplete="off"
+          value={nameInput}
+        />
+        {filteredName.length > 0 ? <ErrorMessage>Achtung! Der eingegebene Name existiert bereits!</ErrorMessage> : null}
+        {isName ? <ErrorMessage>Bitte gebe einen Namen ein!</ErrorMessage> : null}
+        {nameInput.length >= 20 ? <ErrorMessage>Die maximale Eingabe an Zeichen ist erreicht!</ErrorMessage> : null}
+
+        <LabelDate htmlFor="birthdate">Wann ist Dein Kind geboren?</LabelDate>
+        <DayPicker
+          id="birthdate"
+          name="birthdate"
+          selected={startDate}
+          onChange={date => {
+            if (date === null) {
+              setStartDate(new Date());
+            } else {
+              setStartDate(date);
+            }
+          }}
+          dateFormat="dd.MM.yyyy"
+          showYearDropdown
+          dropdownMode="select"
+          maxDate={new Date()}
+          onFocus={handleEmptyNameOnFocusDate}
+        />
+        {startDate === null ? <ErrorMessage>Bitte wähle ein Geburtsdatum aus!</ErrorMessage> : null}
+        {filteredNameAndDate.length > 0 ? (
+          <ErrorMessage>Dein Kind existiert bereits. Eine doppelte Eingabe ist nicht möglich!</ErrorMessage>
+        ) : null}
+
+        <AddButton type="submit" disabled={disabledButton}>
+          <AddIcon fill={disabledButton ? '#19337a' : '#fff'} height="30px" width="30px" />
+          <p disabled={disabledButton}>Erstelle ein Kind</p>
```

Comment:
This `disabled` doesn't do anything on this `p` element. Also, we usually don't use block elements within inline elements. I would probably use a `span` here.
```suggestion
          <span>Erstelle ein Kind</span>
```

Reply:
thx